### PR TITLE
Update TPC QA

### PR DESCRIPTION
**Pull request by njacazio**

### CODE REVIEW

**Comment by njacazio** [comment]
@AnnalenaKa could you please also have a look? Thanks

File: Common/TableProducer/PID/pidTPC.cxx [hunk 1]
```diff
@@ -135,7 +136,8 @@ struct tpcPid {
     }
   }
 
-  void process(Coll const& collisions, Trks const& tracks)
+  void process(Coll const& collisions, Trks const& tracks,
+               aod::BCsWithTimestamps const&)
```
Comment (AnnalenaKa) [inline Common/TableProducer/PID/pidTPC.cxx:140]: Where is the Timestamp used later in the task? Or do we need to implement this?

File: Common/TableProducer/PID/pidTPCFull.cxx [hunk 2]
```diff
@@ -317,26 +334,208 @@ struct tpcPidFullQa {
...
+  Configurable<int> nBinsDelta{"nBinsDelta", 200, "Number of bins for the Delta"};
+  Configurable<float> minDelta{"minDelta", -1000.f, "Minimum Delta in range"};
+  Configurable<float> maxDelta{"maxDelta", 1000.f, "Maximum Delta in range"};
+  Configurable<int> nBinsExpSigma{"nBinsExpSigma", 200, "Number of bins for the ExpSigma"};
+  Configurable<float> minExpSigma{"minExpSigma", 0.f, "Minimum ExpSigma in range"};
+  Configurable<float> maxExpSigma{"maxExpSigma", 200.f, "Maximum ExpSigma in range"};
+  Configurable<int> nBinsNSigma{"nBinsNSigma", 200, "Number of bins for the NSigma"};
+  Configurable<float> minNSigma{"minNSigma", -10.f, "Minimum NSigma in range"};
+  Configurable<float> maxNSigma{"maxNSigma", 10.f, "Maximum NSigma in range"};
+  Configurable<int> applyEvSel{"applyEvSel", 0, "Flag to apply rapidity cut: 0 -> no event selection, 1 -> Run 2 event selection, 2 -> Run 3 event selection"};
```
Comment (AnnalenaKa) [inline Common/TableProducer/PID/pidTPCFull.cxx:402]: In line 204 you have set the event selection to 2, so Run 3 and here it is kept at 0, so no event selection. Is this done on purpose for the tpcPidFullQaWTof?

File: Common/TableProducer/PID/pidTPC.cxx (hunk 1 above)
Comment (njacazio) [reply Common/TableProducer/PID/pidTPC.cxx:140]: I will add it in another PR, I wanted to set this first, this is what I mentioned to you yesterday that is missing

File: Common/TableProducer/PID/pidTPCFull.cxx (hunk 2 above)
Comment (njacazio) [reply Common/TableProducer/PID/pidTPCFull.cxx:402]: I will make it the same

File: Common/TableProducer/PID/pidTPC.cxx (hunk 1 above)
Comment (AnnalenaKa) [reply Common/TableProducer/PID/pidTPC.cxx:140]: Ah I see. This answers my question thanks!

File: Common/TableProducer/PID/pidTPCFull.cxx (hunk 2 above)
Comment (njacazio) [reply Common/TableProducer/PID/pidTPCFull.cxx]: Done ;)

**Comment by AnnalenaKa** [comment]
> 

The additional QA looks good to me. Thanks for adding the various QA histograms and the option with the TOF as well!

**Comment by njacazio** [comment]
Yes, I need it myself so ;)

**Review by iarsene — APPROVED** [review]

**Pull request merged**